src/openeo_odc/map_processes_odc.py

```diff
@@ -46,20 +46,24 @@ def map_load_collection(id, process):
                     highLon     = np.max([[el[0] for el in polygon[0]]])
                     params['x'] = (lowLon,highLon)
                     params['y'] = (lowLat,highLat)
+    params['time'] = []
     if 'temporal_extent' in process['arguments']:
         def exclusive_date(date):
-            return str(np.datetime64(date) - np.timedelta64(1, 'D')).split(' ')[0] # Substracts one day
+            return str(np.datetime64(date) - np.timedelta64(1, 'D')) + 'Z' # Substracts one day
         if process['arguments']['temporal_extent'] is not None:
-            timeStart = '1970-01-01'
-            timeEnd   = str(datetime.now()).split(' ')[0] # Today is the default date for timeEnd, to include all the dates if not specified
-            if process['arguments']['temporal_extent'][0] is not None:
+            if len(process['arguments']['temporal_extent'])>0:
+                timeStart = '1970-01-01'
+                timeEnd   = str(datetime.now()).split(' ')[0] # Today is the default date for timeEnd, to include all the dates if not specified
+                if process['arguments']['temporal_extent'][0] is not None:
                 timeStart = process['arguments']['temporal_extent'][0]
             if process['arguments']['temporal_extent'][1] is not None:
                 timeEnd = process['arguments']['temporal_extent'][1]
             params['time'] = [timeStart,exclusive_date(timeEnd)] 
     if 'crs' in process['arguments']['spatial_extent']:
         params['crs'] = process['arguments']['spatial_extent']['crs']
+    params['measurements'] = []
     if 'bands' in process['arguments']:
+        if len(process['arguments']['bands'])>0:
```

Comment:
Identation error on the next line

Reply:
I think in this case the if on the line 65 and 66 could be combined, or?

Reply:
Yes, same also for temporal_extent